pipelines/broad/reprocessing/external/exome/test_inputs/Plumbing/RP-929.NA12878.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ExternalExomeReprocessing.input_cram": "gs://broad-gotc-test-storage/germline_single_sample/exome/plumbing/truth/develop/RP-929.NA12878/NA12878_PLUMBING.cram",
+  "ExternalExomeReprocessing.input_cram": "https://github.com/sbg/sbg_dockstore_tools/raw/master/topmed-workflows/variant-caller/testdata/NA12878_plumbing.cram",
```

Comment:
@mr-c just an update this has not been forgotten!! we had to do some brainstorming about the correct location and the ability to be public. hoping to have this ready for you early next week. I also think once we have the link we would like to add an example_inputs file (just a copy of `RP-929.NA12878.json` with a different name) next to the test_inputs directory. The new file can include the update with the public link I'll provide. Since we run our smart tests in the test_inputs directory I don't think we want it changed from using the `broad-gotc-test-storage` path. 

Please let me know if this doesn't make sense and I can suggest further changes! But no need to do anything until I get you the correct link. Thank you for your patience!!

Reply:
Thanks for the update @cheyenne-gold ; as long as there is a well-marked inputs JSON with public data, that works for me! No reason to drop your other sources.

Reply:
Okay, the cram is available here: `gs://broad-public-datasets/NA12878/germline_single_sample/exome/plumbing/RP-929.NA12878/NA12878_PLUMBING.cram`!! Thank you again for your patience! Can you please revert the change in this file and then create an `example_inputs.json` next to the test_inputs directory? The update to the public bucket can live there! There are other examples of this approach throughout WARP, but please let me know if you need any assistance!!